client/js/screens/UserProfile/UserProfile.js

```diff
@@ -0,0 +1,117 @@
+          <Text style={styles.locationStatus}>
+            <Ionics name={`ios-today`} size={25} /> Vancouver, Bc
+          </Text>
+          <Text style={styles.locationStatus}>
+            <Ionics name={`ios-book`} size={25} /> Burnaby High School
+          </Text>
+          <View style={styles.interest}>
+            <View style={styles.section}>
+              <Text style={styles.sectionTitle}>About Me</Text>
+              <Text style={styles.sectionContent}>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
+                vitae porta magna. Lorem ipsum dolor sit amet. Lorem ipsum dolor
+                sit amet, consectetur adipiscing elit. Cras vitae porta magna.
+                Lorem ipsum dolor sit amet.
+              </Text>
+            </View>
+            <View style={styles.section}>
+              <Text style={styles.sectionTitle}>Favorite Ways to Meet</Text>
+              {/* TODO loop icons */}
+              <View style={styles.sectionContent}>
+                <View style={styles.icon}>
+                  <Ionics name={`ios-cafe`} size={25} />
+                  <Text>Coffe</Text>
+                </View>
+                <View style={styles.icon}>
+                  <Ionics name={`ios-school`} size={25} />
+                  <Text>After School</Text>
+                </View>
+                <View style={styles.icon}>
+                  <Ionics name={`ios-walk`} size={25} />
+                  <Text>Walk</Text>
+                </View>
+              </View>
+            </View>
+            <View style={styles.section}>
+              <Text style={styles.sectionTitle}>Personal Interests</Text>
+              {/* TODO loop globes */}
+              <View style={styles.sectionContent}>
+                <Text style={styles.globe}>Sports</Text>
+                <Text style={styles.globe}>Dance</Text>
+                <Text style={styles.globe}>ETC</Text>
+              </View>
+            </View>
+            <View style={styles.section}>
+              <Text style={styles.sectionTitle}>Social Interests</Text>
+              {/* TODO loop globes */}
+              <View style={styles.sectionContent}>
+                <Text style={styles.globe}>Sports</Text>
+                <Text style={styles.globe}>Dance</Text>
+                <Text style={styles.globe}>ETC</Text>
+              </View>
+            </View>
+            <View style={styles.section}>
+              <Text style={styles.sectionTitle}>Profesional Intereset</Text>
+              {/* TODO loop globes */}
+              <View style={styles.sectionContent}>
+                <Text style={styles.globe}>Sports</Text>
+                <Text style={styles.globe}>Dance</Text>
+                <Text style={styles.globe}>ETC</Text>
+              </View>
+            </View>
+          </View>
+        </View>
+      </ScrollView>
+    );
+  }
+}
+
+export default withNavigation(UserProfile);
```

Comment:
instead of using withNavigation you can pass navigation object as a prop from your container file or export `userProfile` from index.js of this dir.
```
// client/js/screens/UserProfile/index.js

import UserProfileContainer from './UserProfileContainer'
import UserProfile from './UserProfile'

export {UserProfile}
export default UserProfileContianer 
```

Use the same convention for all of your screens.